### DISCUSSION TITLE
Use symbolic icon in system tray

### DISCUSSION THREAD

**Discussion by adrianinsaval** [rustdesk/rustdesk#9239] [General]
Rustdesk's icon is somewhat disruptive on a system tray, not hugely since there's plenty of apps that also don't use symbolic icons but it would be nice if rustdesk could use symbolic icons for the system tray.
![image](https://github.com/user-attachments/assets/34f9d84f-77e4-4d0d-8240-306501d70dca)

**Comment by rksdutt** [comment]
It would be great if Rustdesk could use symbolic icons for the system tray. 
I hope the development team considers this for future updates to improve consistency and user experience."

**Reply by benbc7** [reply]
![rustdesk](https://github.com/user-attachments/assets/13eeabf1-4325-432b-b060-7da5fba05fd6)

I even tried making it, hoping I could just add it to my icon theme and change it, but unfortunately not.

**Comment by wodin** [comment]
![Screenshot of icons in MATE Indicator Applet](https://github.com/user-attachments/assets/513cbc36-06d2-41a7-b9a9-b4660c119314)

This is what it looks like for me on the MATE Indicator Applet in dark mode
